src/ipc/bridge.cc

```diff
@@ -2616,6 +2634,9 @@ static void registerSchemeHandler (Router *router) {
     if (result.post.event_stream != nullptr) {
       *result.post.event_stream = [task](const char* name, const char* data,
                                          bool finished) {
+        if (![_tasks containsObject: task]) {
+          return false;
```

Comment:
Callers of `sapi_ipc_send_event` or send_chunk should stop their work if `false` is returned.